SETUP/tests/manual_web/split_test/switchable_split.js

```diff
@@ -41,7 +41,8 @@ $(function () {
     }
 
     let initialSplitVertical = false;
-    let mainSplit = splitControl("#container", { splitVertical: initialSplitVertical });
+    const container = document.getElementById("container");
```

Comment:
*minor*, I would just inline it with `splitControl(document.getElementById('...`. This happens the other instances too if you decide you want to make this change.

Reply:
Yes, good idea.